### PR TITLE
Added Beverage Jugs to Lathe

### PR DESCRIPTION
**Pull request by SleepyScarecrow**
# Description

This adds the Beverage Jugs for the bartender back into the autolathe, as to make them easier to get a hold of for things like juices.

---

# Changelog

:cl: Tilkku
tweak: added beverage jugs to autolathe

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml [hunk 1]
```diff
@@ -161,6 +161,7 @@
       - WeaponCapacitorRechargerCircuitboard
       - HandheldStationMap
       - ClothingHeadHatWelding
+      - CustomDrinkJug
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Structures/Machines/lathe.yml:164]:
```suggestion
      - CustomDrinkJug # FloofStation
```

File: Resources/Prototypes/Recipes/Lathes/cooking.yml [hunk 2]
```diff
@@ -97,3 +97,10 @@
   completetime: 0.4
   materials:
     Steel: 100
+
+- type: latheRecipe
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Recipes/Lathes/cooking.yml:101]:
```suggestion
- type: latheRecipe # FloofStation
```

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
Also please follow the correct Format for the PR, Aka a correct description and changlogs.

Common ya did it before!

**Comment by SleepyScarecrow** [comment]
In my defense I did pr this while in shift

**Review by Fansana — APPROVED** [review]

**Pull request merged**